acceptance/ui/features/data/default/services.json

```diff
@@ -12,9 +12,31 @@
 
     "childServiceWithVHost": {
       "name": "s2",
-      "vhostName": "websvc",
+      "vhostName" : "websvc",
       "publicPortNumber": 1234,
+      "host"      : "%{local_ip}",
+
+      "endpoint_https"  : "websvc",
+      "protocol_https"  : "HTTPS",
+      "protocol_https_o"  : "HTTPS",
```

Comment:
The input and output are of different strings.. 
Probably should be renamed to _https_in and _https_out..